cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -157,6 +157,31 @@ Feature: Users can authneticate with OIDC authenticator
     Errors::Authentication::AuthnOidc::AdminAuthenticationDenied
     """
 
+  Scenario: Provider uri Dynamic change
+    Given I get authorization code for username "alice" and password "alice"
+    And I fetch an ID Token
+    When I authenticate via OIDC with id token
+    Then user "alice" is authorized
+    # Update provider uri to an unreachable hostname
+    When I add the secret value "http://unreachable.com/" to the resource "cucumber:variable:conjur/authn-oidc/keycloak/provider-uri"
```

Comment:
all the steps above this one should be given. you start the actual test here.

Reply:
Done :)